src/Content/MailArchive/MailArchiveEntity.php

```diff
@@ -26,11 +27,6 @@ class MailArchiveEntity extends Entity
 
     protected ?string $transportState;
```

Comment:
as you want to release a 3.0.0. We should maybe get rid of all methods here and use public properties instead.

Reply:
`dal:validate` says: NO